.circleci/config.yml

```diff
@@ -880,6 +880,31 @@ jobs:
       - run: make docker-perf IMAGE_NAME=$PERF_IMAGE_NAME
       - run: IMAGE_NAME=$PERF_IMAGE_NAME sh .circleci/ci_publish.sh
 
+  run_perf_cli_smoke_test:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      PERF_IMAGE_NAME: "omisego/perf:ci"
```

Comment:
where is generating the `:ci` tagged image?? Also when will this tag get updated?

Reply:
it's a name for the temporary image used only in circle ci.
Generation is here - https://github.com/omgnetwork/elixir-omg/pull/1762/files/4ee5857447db0e44a4b6c70663d41ab015d0d902#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R901

Reply:
ah my bad....it is just local image where I somehow thought it was from remote.

Reply:
what is true is the convention is to name the `current` image - `latest` :)